BUG: Convert output type in Excel for MultiIndex with period levels (#60182)
pandas-dev/pandas: doc/source/whatsnew/v3.0.0.rst

- Bug in :meth:`DataFrame.to_stata` when writing :class:`DataFrame` and ``byteorder=`big```. (:issue:`58969`)
- Bug in :meth:`DataFrame.to_stata` when writing more than 32,000 value labels. (:issue:`60107`)
- Bug in :meth:`DataFrame.to_string` that raised ``StopIteration`` with nested DataFrames. (:issue:`16098`)

> - Bug in :meth:`ExcelFormatter._format_hierarchical_rows` where output type in excel for multiIndex with period levels is not a date (:issue:`60099`)

Reviewer: ```suggestion
- Bug in :meth:`DataFrame.to_excel` where the a :class:`MultiIndex` index with a period level was not a date (:issue:`60099`)
```

Author: Thanks a lot for your suggestion!
